src/chat/narrowsSelectors.js

```diff
@@ -93,7 +93,17 @@ export const getRecipientsInGroupNarrow: Selector<(User | RealmBot)[], Narrow> =
     }),
 );
 
-export const getStreamInNarrow = (narrow: Narrow) =>
+export const getStreamInNarrow = (
+  narrow: Narrow,
+): Selector<{
+  description: string,
+  history_public_to_subscribers: boolean,
+  in_home_view: boolean,
+  invite_only: boolean,
+  is_announcement_only: boolean,
+  name: string,
+  stream_id: number,
+}> =>
```

Comment:
Yeah, I thought it would return Subscription as well but it is missing a bunch of fields that are defined in Subscription but not in Stream and this is resulting in a problem. I will look into it and see what is going on on the inside :smiley: .